datanode/src/storage_api.py

```diff
@@ -76,12 +76,11 @@
 # VERSION = 'PublicPortal1.1.1.006'  # Added public portal support
 VERSION = 'PublicPortal1.1.1.007'  # Fixed multi-cell bug
```

Comment:
New minor version for non-breaking API additions?

Reply:
Done.